src/otx/engine/engine.py

```diff
@@ -323,9 +323,42 @@ def predict(
             return_predictions=return_predictions,
         )
 
-    def export(self, *args, **kwargs) -> None:
-        """Export the trained model to OpenVINO Intermediate Representation (IR) or ONNX formats."""
-        raise NotImplementedError
+    def export(self, checkpoint: str | Path | None = None, export_config: ExportConfig | None = None, **kwargs) -> Path:
```

Comment:
How about using `export_format` and `precision` as argument instead of `ExportConfig`? If then, user can set argument in CLI.

Reply:
Agree with Eunwoo, we can then delete ExportConfig DTO

Reply:
Done